builtin/aws/lambda/platform_logs.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/hashicorp/waypoint-plugin-sdk/component"
 	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	"github.com/hashicorp/waypoint/builtin/aws/utils"
+	"github.com/pkg/errors"
```

Comment:
Nit: this should go into the new line separated section above this because its not a waypoint import.